x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/api/util/prebuilt_rules.ts

```diff
@@ -43,7 +44,7 @@ export interface PrebuiltRulesResults {
  * @param rulesIds The list of IDs to filter requested prebuilt rules. If not specified, all available prebuilt rules will be returned.
  * @returns
  */
-export const getPrebuiltRules = async (
+export const getPrebuiltRulesByIds = async (
```

Comment:
Not sure about this rename, `ruleIds` to be an optional parameter

Reply:
will switch it back 👍